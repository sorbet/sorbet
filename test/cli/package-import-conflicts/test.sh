cd test/cli/package-import-conflicts || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --max-threads=0 . 2>&1
