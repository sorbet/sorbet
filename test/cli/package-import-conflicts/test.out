a/__package.rb:4: Cannot export `A::B` because it is owned by another package https://srb.help/3721
     4 |  export A::B
          ^^^^^^^^^^^
    ab/ab.rb:3: Defined here
     3 |module A::B
        ^^^^^^^^^^^

ab/ab.rb:3: `A::B` was previously defined as a `class` https://srb.help/4012
     3 |module A::B
        ^^^^^^^^^^^
    a/a.rb:4: Previous definition
     4 |  class B; end
          ^^^^^^^

a/a.rb:4: Class or method definition must match enclosing package namespace `A` https://srb.help/3713
     4 |  class B; end
                ^
  Note:
    Constant `A::B` should either be defined in directory `./a/` to match its package, or re-namespaced to be within `A`
Errors: 3
